.github/workflows/CI.yml

```diff
@@ -46,6 +48,8 @@ jobs:
       - uses: actions/checkout@v4
       - name: Setup Bun
         uses: oven-sh/setup-bun@v1
+        with:
+          bun-version: 1.0.26
```

Comment:
since we have trouble upgrading atm, can we land this with the old bun version? we can upgrade on another commit.

Reply:
this is the old version, the new one that has problems is `1.1.6`